.infra/crons.ts

```diff
@@ -106,4 +106,8 @@ export const crons: Cron[] = [
     schedule: '*/5 * * * *',
     activeDeadlineSeconds: 4 * 60,
   },
+  {
+    name: 'calculate-top-readers',
+    schedule: '0 2 1 * *',
```

Comment:
First of the month at 02:00.

The cron `update-highlighted-views` that runs at 04:15 each day handles the refresh of the UserStats MV, so no point really in running it in this cron.